### ISSUE TITLE
predict insertions

### ISSUE THREAD

**Issue by trudack** [ryanmcgreevy/ModelMaker#32]
In corntarst to the new rosetta_scripts used for ab initio prediction the rosseta tools used for insertions does not keep the chain IDs and segnames. The first chain is A the second is B and all segnames are deleted.

I would suggest to include an automated renaming of segnames and chain IDs within the analysis module if -insertion yes is set. You can get the segname and chain ID from the template file and then give the correct segnames and chain IDs for the aligned predicted structures in the folder pdb_aligned.

**Comment by ryanmcgreevy** [comment]
Assuming the chain ID conversion is 1:1, it is easy enough to change them back. However, how would you go about putting back in segnames when they were all deleted? You can't go by the index of the atom from the template, since you've inserted atoms and the number is now affected. Maybe there is just something I'm missing here...

**Comment by trudack** [comment]
Yes the chain conversion is 1:1 so first change chain A to the first chain ID chain B to the second and so on. After renaming the chains you can rename the segnames based on the chain IDs of the template set chain ID 1 to segname 1 of template

**Comment by ryanmcgreevy** [comment]
This assumes though that the chains and segnames selections are exactly the same, which isn't necessarily the case (I think).

**Comment by ryanmcgreevy** [comment]
For future reference, @trudack and I discussed this on Skype and ultimately the way to solve this, which should probably be applied to every command that calls Rosetta, is to use the sequence and the input pdb to figure out and fix first the resid's so they are the same, then use those to fix the segnames and chains.
